lib/Auth/Source/Globus.php

```diff
@@ -92,20 +92,25 @@ protected function samlize($array, $prefix = '') {
     protected function getFullName($name) {
```

Comment:
this seems overly complicated compared to something like this:
```php
function getFullName($name){
    $trimmedName = trim($name);
    $lastName = 'UNKNOWN';
    $firstName = 'UNKNOWN';
    if(strpos($trimmedName, ',') !== false ){
        list($lastName, $firstName) = explode(',', $trimmedName, 2);
    }
    else if(strpos($trimmedName, ' ') !== false ){
        list($firstName, $lastName) = explode(' ', $trimmedName, 2);
    }
    return array(
        'first_name' => trim($firstName),
        'last_name' => trim($lastName)
    );
}
```

old:
```
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => 
    [last_name] => 
)
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => First
    [last_name] => Last
)
Array
(
    [first_name] => First
    [last_name] => Middle Last
)
Array
(
    [first_name] => First
    [last_name] => von Last
)
Array
(
    [first_name] => First
    [last_name] => M. Last
)
Array
(
    [first_name] => First
    [last_name] => M. von Last
)
Array
(
    [first_name] => First
    [last_name] => Last
)
Array
(
    [first_name] => First Middle
    [last_name] => Last
)
Array
(
    [first_name] => First
    [last_name] => von Last
)
Array
(
    [first_name] => First M.
    [last_name] => Last
)
Array
(
    [first_name] => First M.
    [last_name] => von Last
)
```
new:
```
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => 
    [last_name] => 
)
Array
(
    [first_name] => UNKNOWN
    [last_name] => UNKNOWN
)
Array
(
    [first_name] => First
    [last_name] => Last
)
Array
(
    [first_name] => First
    [last_name] => Middle Last
)
Array
(
    [first_name] => First
    [last_name] => von Last
)
Array
(
    [first_name] => First
    [last_name] => M. Last
)
Array
(
    [first_name] => First
    [last_name] => M. von Last
)
Array
(
    [first_name] => First
    [last_name] => Last
)
Array
(
    [first_name] => First Middle
    [last_name] => Last
)
Array
(
    [first_name] => First
    [last_name] => von Last
)
Array
(
    [first_name] => First M.
    [last_name] => Last
)
Array
(
    [first_name] => First M.
    [last_name] => von Last
)
```